packages/mariadb/mariadb.0.5.0/opam

```diff
@@ -15,5 +15,12 @@ depends: [
   "ocamlfind" {build}
   "ctypes" {>= "0.7.0"}
   "ctypes-foreign" {>= "0.4.0"}
+  "async"
+  "lwt"
```

Comment:
Yes. It builds its example files, which include both Async examples and Lwt examples.